docsite/docs.css

```diff
@@ -59,14 +49,13 @@ aside>h1::after{
 .active{
     color:var(--primary-color);
     font-size: 1.4rem;
-    font-weight: bolder;
+    font-weight: bold;
```

Comment:
try to use 700, 500 numbers instead of `bold` text values

Reply:
Done!!